spec/features/move_locations_spec.rb

```diff
@@ -23,7 +23,7 @@
     expect do
       fill_in "User swipe card id/barcode", with: user.swipe_card_id
       fill_in "New location barcode (Parent location)", with: parent_location.barcode
-      fill_in "Location barcodes to be moved (Child location)", with: child_locations.join_barcodes + "\nlw-no-location-here"
+      fill_in "Location barcodes to be moved (Child location)", with: "#{child_locations.join_barcodes} \nlw-no-location-here"
```

Comment:
Think you've added a space here, after `#{child_locations.join_barcodes}`
Probably doesn't matter